toml/src/main/kotlin/org/rust/toml/annotator/CargoTomlErrorAnnotator.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.annotator
+
+import com.intellij.ide.annotator.AnnotatorBase
+import com.intellij.lang.annotation.AnnotationHolder
+import com.intellij.psi.PsiElement
+import org.rust.toml.CargoTomlDependencyVisitor
+import org.rust.toml.crates.getCrate
+import org.toml.lang.psi.TomlKey
+import org.toml.lang.psi.TomlValue
+
+class CargoTomlErrorAnnotator : AnnotatorBase() {
+    override fun annotateInternal(element: PsiElement, holder: AnnotationHolder) {
+        object : CargoTomlDependencyVisitor() {
+            override fun visitDependency(name: TomlKey, version: TomlValue?) {
+                checkDependency(name, version, holder)
+            }
+        }.visitElement(element)
+    }
+
+    private fun checkDependency(dependency: TomlKey, version: TomlValue?, holder: AnnotationHolder) {
+        val crate = getCrate(dependency)
+
+        if (crate == null) {
```

Comment:
If we don't have an internet connection, we can't make request to crates.io. As a result, we get `null` here and provide false-positive annotation
![image](https://user-images.githubusercontent.com/2539310/91032308-7417ce00-e60a-11ea-84b1-8d6ce78894c5.png)